optimum/exporters/openvino/model_configs.py

```diff
@@ -131,6 +132,10 @@ def init_model_configs():
         "transformers",
         "Qwen2VLForConditionalGeneration",
     )
+    TasksManager._CUSTOM_CLASSES[("pt", "qwen2-5-vl", "image-text-to-text")] = (
+        "transformers",
+        "Qwen2_5_VLForConditionalGeneration",
+    )
```

Comment:
why not use `AutoModelForImageTextToText` ? https://github.com/huggingface/transformers/blob/v4.49.0/src/transformers/models/auto/modeling_auto.py#L836

Reply:
UPD: it seems that task manager does not allow to specify multiple loaders classes in case of remote code models, where there is no model in transformers explicitly. I'm investigating if it can be updated somehow (e.g. based auto_mapping from config instead of loaded library)

Reply:
thanks @eaidova, merging for now and we can still update this in a following PR